src/components/Books.js

```diff
@@ -14,4 +14,5 @@ export const booksColumns = [
   { title: "System", field: "system" },
   { title: "Key", field: "key", grouping: false },
   { title: "Initials", field: "initials", grouping: false },
+  { title: "Publisher", field: "publisher", grouping: false },
```

Comment:
If we do include this, grouping would seem useful.

Reply:
Valid remark. I will follow up and if action is required, I will add the grouping with the next commit.